tests/SampleApps/nodejs/monorepo-lerna-yarn/lerna.json

```diff
@@ -0,0 +1,8 @@
+{
+  "npmClient": "yarn",
```

Comment:
Does `lerna` take care of choosing either `npm` or `yarn`?

Reply:
The default is client is npm, Lerna can choose which one by having this line in `lerna.json` file.